documentation/source/user_section/tutorials/diffusion-weighted-mri/preprocessing.rst

```diff
@@ -33,19 +33,19 @@ The resulting 3D volume can then be used to obtain a cord segmentation.
 
 .. code::
 
-   sct_deepseg_sc -i dmri_dwi_mean.nii.gz -c dwi -qc ~/qc_singleSubj
+   sct_deepseg -task seg_sc_contrast_agnostic -i dmri_dwi_mean.nii.gz -qc ~/qc_singleSubj
```

Comment:
re: e30aa9cbbf9c13abcb3384770e83d8f28a685e96; this change is currently broken by #4805.

Reply:
WIP fix here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4809